src/main/java/com/kk/strong/controller/ExerciseController.java

```diff
@@ -36,8 +35,13 @@ public ResponseEntity<?> updateExercise(@PathVariable Long exerciseId, @RequestB
     }
 
     @DeleteMapping("/{exerciseId}")
-    public ResponseEntity<?> deleteExercise(@PathVariable Long exerciseId){
+    public ResponseEntity<?> deleteExercise(@PathVariable Long exerciseId) {
         exerciseService.deleteExercise(exerciseId);
         return ResponseEntity.noContent().build();
     }
+
+    @GetMapping("/user_progress/{userId}/{exerciseType}")
```

Comment:
I would prefer the path to be something like `/users/{userId}/exercises/{exerciseType}`, but I would need to either put it in UserController, remove RequestMapping("/exercises") for this controller or put it in a new controller, I think the last option would be preffered, thoughts?